samples/nrf9160/modem_shell/prj.conf

```diff
@@ -128,6 +129,9 @@ CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS=17
 
 # Cellular positioning services
 CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+# Uncomment the following if MQTT is selected, for example when CONFIG_MOSH_CLOUD is set. Otherwise,
+# REST API is used for communication between the device and nRF Cloud.
+#CONFIG_NRF_CLOUD_CELL_POS=y
```

Comment:
Would it make sense to select this automatically in Kconfig if MOSH_CLOUD and MULTICELL_LOCATION_SERVICE_NRF_CLOUD are enabled?

Reply:
Agreed